perseo-main/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>perseo-core</artifactId>
         <groupId>com.telefonica.iot</groupId>
-        <version>1.11.0-SNAPSHOT</version>
+        <version>1.12.0</version>
```

Comment:
perseo-utils in this pom.xml file has to be updated also

Reply:
Done in https://github.com/telefonicaid/perseo-core/pull/211/commits/dd9b9dd07053239a79bfd0eabef195d86f48f141